src/components/invoice/invoice-edit/EditInvoice.tsx

```diff
@@ -114,6 +134,12 @@ export class EditInvoice extends Component<EditInvoiceProps, EditInvoiceState> {
     this.forceUpdate();
   }
 
+  displayBadge(label: string) {
+    return (
+      <Badge style={{marginLeft: 10, fontSize: '100%', fontWeight: 300}} variant="secondary">{label}</Badge>
+    );
+  }
```

Comment:
Dit is echt een anti-pattern in React. 
Hier valt het natuurlijk nog goed mee maar als je hierop voortbouwt dan wordt na een tijdje `this.xxx` gebruikt in de `displayBadge` en vanaf dan beginnen de problemen.

Op het eerste project waar ik op zat met React werd dit ook veel gedaan -- dit eindigt altijd in tranen 😃


Prefereer altijd Componenten. Compose de Componenten. De Componenten zijn functies, je hoeft er niet nog een functie rond te schrijven. Vb:

```
const EditInvoiceBadge = ({label}) => <Badge style={{marginLeft: 10, fontSize: '100%', fontWeight: 300}} variant="secondary">{label}</Badge>
```